### DISCUSSION TITLE
API Get Device Info

### DISCUSSION THREAD

**Discussion by priramdona** [aldinokemal/go-whatsapp-web-multidevice#197] [Q&A]
Numpang tanya sifu, untuk mendapatkan Device Information ketika sukses login adakah API nya?
![pic1](https://github.com/user-attachments/assets/d3225abc-0a36-4f35-9fad-b79c2fa60879)

di Documentation tertera mulai dari /app/login..

**Comment by aldinokemal** [comment]
coba bisa pake ini, GET /app/devices
